base/setup/reactos/reactos.h

```diff
@@ -108,6 +111,15 @@ typedef struct _NT_WIN32_PATH_MAPPING_LIST
 } NT_WIN32_PATH_MAPPING_LIST, *PNT_WIN32_PATH_MAPPING_LIST;
 
 
+#if 0
+typedef struct _KBLAYOUT
+{
+    TCHAR LayoutId[9];
+    TCHAR LayoutName[128];
+    TCHAR DllName[128];
+} KBLAYOUT, *PKBLAYOUT;
+#endif
+
```

Comment:
Why did you move that typedef to the bottom? Can you move it back up to aovid the useless diff or wouldn't that compile anymore?